frameworks/angular/container/src/app/view2/view2.component.ts

```diff
@@ -1,42 +1,66 @@
-import { Component, NgZone } from '@angular/core';
+import "@finos/fdc3";
```

Comment:
You shouldn't need to import the fdc3 npm package script as we inject the fdc3 api if you specify it in default view options or view options of the specific. view.

Reply:
Removed